wpilibc/src/main/native/cpp/RobotBase.cpp

```diff
@@ -65,7 +83,30 @@ class WPILibCameraServerShared : public frc::CameraServerShared {
 }  // namespace
 
 static void SetupCameraServerShared() {
-  SetCameraServerShared(std::make_unique<WPILibCameraServerShared>());
+#ifdef __FRC_ROBORIO__
+  auto cameraServerLib = dlopen("libcameraserver.so", RTLD_NOW);
+  if (!cameraServerLib) {
+    cameraServerLib = dlopen("libcameraserverd.so", RTLD_NOW);
```

Comment:
This is definitely a change from current behavior, where if you deploy debug you'll get debug and if you deploy release you'll get release.  It's easy to end up with both libraries installed.

Reply:
Oh yea, good point. I'll figure something out for that.